src/Resources/views/storefront/page/search/search-pagelet.html.twig

```diff
@@ -2,58 +2,23 @@
 
 {% set listingColumns = 'col-sm-6 col-lg-4 col-xl-3' %}
```

Comment:
When we add all those other variables inside the enabled check, should we also move this there?

Reply:
Or why is this even needed? If it is something from Shopware, I guess this should also work without us setting it again

Reply:
Seems like it's not necessary, classes are still set when removed
![Screen Shot 2022-04-07 at 10 08 47](https://user-images.githubusercontent.com/511559/162152011-73cc8706-9fb3-42a7-8dc7-c183d437acb7.png)
: